test/sanity/issue5882-debugging-tools-crash/test.py

```diff
@@ -0,0 +1,72 @@
+import time
+import os
+import shutil
+import subprocess
+import platform
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from nw_util import *
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys 
+from selenium.webdriver.support.select import Select
+
+chrome_options = Options()
+testdir = os.path.dirname(os.path.abspath(__file__))
+chrome_options.add_argument("nwapp=" + testdir)
+node_module = os.path.join(testdir, "node_modules")
+os.chdir(testdir)
+
+try:
+    shutil.rmtree(node_module)
+except:
+    pass
+
+assert(False == os.path.isdir(node_module))
+
+if platform.system() == "Windows":
+    subprocess.call(["npm", "install", "request"], shell=True)
+else:
+    subprocess.call(["npm", "install", "request"])
```

Comment:
should use the util function to install the module.

Reply:
Updated